geoips/plugins/modules/readers/viirs_netcdf.py

```diff
@@ -47,18 +47,21 @@
 (discussion with Mindy on how to do it).
 """
 # Python Standard Libraries
+from collections import defaultdict
+from datetime import datetime
 import logging
 import os
 
-# Installed Libraries
+# Third-Party Libraries
 import numpy
 import pandas as pd
 import xarray as xr
 
-from geoips.utils.context_managers import import_optional_dependencies
-
 # GeoIPS Libraries
```

Comment:
Why was this section not changed as the others were? Was this process done manually?

Reply:
This is fixed now. Yes.